src/display/api.js

```diff
@@ -104,10 +104,16 @@ if (typeof PDFJSDev !== 'undefined' &&
  * @property {string} docBaseUrl - (optional) The base URL of the document,
  *   used when attempting to recover valid absolute URLs for annotations, and
  *   outline items, that (incorrectly) only specify relative URLs.
- * @property {boolean} disableNativeImageDecoder - (optional) Disable decoding
+ * @property {boolean} disableNativeImageDecoder - (deprecated) Disable decoding
  *   of certain (simple) JPEG images in the browser. This is useful for
  *   environments without DOM image support, such as e.g. Node.js.
  *   The default value is `false`.
+ * @property {string} nativeImageDecoderSupport - (optional) Strategy when
+ *   decoding of certain (simple) JPEG images in the browser.
+ *   The default value is `decode`; `none` for environments without DOM image
+ *   and canvas supports, such as e.g. Node.js; `display` for also environments
+ *   without supports above but add a simple Image object stub to decode JPEG
+ *   images as much as possible, see more in `examples/node/pdf2svg.js`.
```

Comment:
The wording/grammar can probably be improved here, and I don't think that the API documentation should refer to a *specific* example file; e.g. something like this:
```
* @property {string} nativeImageDecoderSupport - (optional) Strategy for
*   decoding certain (simple) JPEG images in the browser. This is useful for
*   environments without DOM image and canvas support, such as e.g. Node.js.
*   Valid values are 'decode', 'display' or 'none'; where 'decode' is intended
*   for browsers with full image/canvas support, 'display' for environments
*   with limited image support through stubs (useful for SVG conversion),
*   and 'none' where JPEG images will be decoded entirely by PDF.js.
*   The default value is 'decode'.
```